package.json

```diff
@@ -1,22 +1,24 @@
 {
-  "name": "@balena/gsheet-utils",
+  "name": "dreamsheets",
```

Comment:
Should this be

`"name": "@balena/dreamsheets",`

?

Reply:
@AlidaOdendaal I don't think so, scoping the package in this way is useful for private packages, and for branding purposes if the product is directly linked to the balena platform. In this case I don't think it's necessary.